podpac/core/coordinates/array_coordinates1d.py

```diff
@@ -233,6 +233,29 @@ def copy(self):
         kwargs = self.properties
         return ArrayCoordinates1d(self.coordinates, **kwargs)
 
+    def simplify(self):
+        """ Get the simplified/optimized representation of these coordinates.
+
+        Returns
+        -------
+        simplified : ArrayCoordinates1d, UniformCoordinates1d
+            UniformCoordinates1d if the coordinates are uniform, otherwise ArrayCoordinates1d
+        """
+
+        from podpac.core.coordinates.uniform_coordinates1d import UniformCoordinates1d
+
+        # if self.is_uniform:
+        #      return UniformCoordinates1d(self.start, self.stop, self.step, **self.properties)
+
+        TOL = 1e-7
+        dcrds = np.diff(self.coordinates)
+        if np.all(np.abs(dcrds - dcrds[0]) < np.abs(TOL * dcrds[0])):
+            return UniformCoordinates1d(
+                self.coordinates[0], self.coordinates[-1], self.coordinates[1] - self.coordinates[0], **self.properties
+            )
+
+        return self
+
```

Comment:
@mpu-creare This is basically `make_coords` from before. Two things:

1. We will be able to use `self.start, self.stop, self.step` after merging #395 
2. We should be able to use `is_uniform` for the check, but that doesn't currently have a tolerance. I think we should add a tolerance to the `is_uniform` check for array coordinates. What do you think?
2b. I bet this check doesn't work for time coordinates.

Reply:
I agree with a tolerance on 2a. 
I wouldn't be surprised if it doesn't work for time -- future optimization. :)

Reply:
ArrayCoordinates1d is_uniform uses allclose, so it already has a tolerance. I used allclose in DependentCoordinates simplify, also.

Reply:
The only thing to be careful of there is conversion between float32 and float64. Files often are saved at float32, and then loaded into memory as float32. I'm not sure `all_close` can detect between those two cases...

Reply:
Okay, I added `atol=1e-7`.

Reply:
Should I also use `rtol=0`?

Reply:
> Should I also use rtol=0?

The answer is yes.